frontend/src/common/entities.ts

```diff
@@ -81,7 +81,7 @@ export interface Dataset {
   dataset_assets: DatasetAsset[];
   processing_status: DatasetUploadStatus;
   collection_id: Collection["id"];
-  collection_visibility: Collection["visibility"];
+  collection_access_type: Collection["access_type"];
```

Comment:
Is collection_access_type a field on dataset? I don't see it on the [corpora-api.yaml](https://github.com/chanzuckerberg/corpora-data-portal/blob/8ad48e003283b328d7bd75ffc44140a6400479b0/backend/config/corpora-api.yml#L533)

Reply:
it's a field on collections. Does it also need to be a field on dataset?

Reply:
I was pulling this value from the collection this dataset is associated with.

Reply:
Ah since the property is on the collection, we don't need to add `collection_access_type` on dataset object, since that implies `dataset.collection_access_type` is available, which it's not!